hermes-api/src/main/java/pl/allegro/tech/hermes/api/EndpointAddress.java

```diff
@@ -17,28 +17,14 @@
 @JsonSerialize(using = EndpointAddressSerializer.class)
 public class EndpointAddress {
 
-    private static final String ANONYMIZED_PASSWORD = "*****";
```

Comment:
i know the functionality remains as a whole, but i want to deploy this on prod (and probably other clients as well) without breaking existing stuff, and we have some basic-auth subs